plugins/workspace-minimap/src/minimap.ts

```diff
@@ -27,44 +27,55 @@ const BlockEvents = new Set([
  * your code looks like, and how it is organized.
  */
 export class Minimap {
-    protected primaryWorkspace: Blockly.WorkspaceSvg;
-    protected minimapWorkspace: Blockly.WorkspaceSvg;
+    private primaryWorkspace: Blockly.WorkspaceSvg;
+    minimapWorkspace: Blockly.WorkspaceSvg;
     private onMouseMoveWrapper: Blockly.browserEvents.Data;
     /**
-     * Constructor for a minimap
-     * @param workspace The workspace to mirror
+     * Constructor for a minimap.
+     * @param workspace The workspace to mirror.
      */
     constructor(workspace: Blockly.WorkspaceSvg) {
       this.primaryWorkspace = workspace;
-      const options = {
-        readOnly: true,
-        move: {
-          scrollbars: {
-            vertical: true,
-            horizontal: true,
-          },
-          drag: false,
-          wheel: false,
-        },
-      };
-      this.minimapWorkspace = Blockly.inject('minimapDiv', options);
     }
 
     /**
      * Initialize.
      */
     init(): void {
+      // Create a wrapper div for the minimap injection.
+      const minimapWrapper = document.createElement('div');
+      minimapWrapper.id = 'minimapWrapper_' + this.primaryWorkspace.id;
+      minimapWrapper.className = 'minimapWrapper';
+
+      // Make the wrapper a sibling to the primary injection div.
+      const developerDiv = this.primaryWorkspace.getInjectionDiv().parentNode;
+      developerDiv.appendChild(minimapWrapper);
+
+      // Inject the minimap workspace
+      this.minimapWorkspace = Blockly.inject(minimapWrapper.id,
```

Comment:
Also can you check that this works properly in RTL? I think you probably need to set the RTL value of the minimap to match the primary workspace. If it's not working that can be fixed in a follow up!

Reply:
I now have it inheriting the RTL value of the primary workspace and it works as intended.